HelpMyStreetFE/HelpMyStreetFE/Views/Shared/Components/PopulatedDataTable/PopulatedDataTable.cshtml

```diff
@@ -0,0 +1,20 @@
+﻿@using HelpMyStreet.Contracts.ReportService;
+@model HelpMyStreetFE.Models.Account.Report.PopulatedDataTableViewModel
+
+@if (Model.Chart.DataPoints.Count > 0)
+{
+    <div class="rTable">
+        <div class="rTableRow">
+            <div class="rTableHead"><strong>@Model.Chart.XAxisName</strong></div>
```

Comment:
`<strong>` and `<span style="font-weight: bold;">` are two ways of doing the same thing.
Better instead to put it in the style sheet...
`.rTableHead {
  font-weight: bold;
}
`